### DISCUSSION TITLE
gns3-remote-install.sh changes hostname

### DISCUSSION THREAD

**Discussion by MrVining** [GNS3/gns3-server#2068] [General]
I started running GNS3 on a Ubuntu 22.04 bare metal install. I noticed gns3-remote-install.sh changed the hostname of the server. 

Is this a requirement? 
Does changing the hostname back require any other changes?

**Comment by ghost** [comment]
Within the linux OS there might be some configuration files, that contain the hostname. I would search within /etc with `sudo grep -riw gns3vm /etc` to locate those files and then change them. 

As far as I know in GNS3 only the IOU license depends on the hostname. If you change the hostname you have to update it.

**Reply by grossmj** [reply]
Correct, the hostname is only required for the IOU license.

**Reply by MrVining** [reply]
Admittedly I haven't spent an extensive amount of time researching this, but I've tried several google searches, looked though the gns3-remote-install.sh script, and I'm still at a loss. How would update the IOU license? I can't even find this file.

**Reply by eantowne** [reply]
IOU license only applies if you are using Cisco IOU images.

IOU is an internal proprietary Cisco virtual machine that is not available for commercial use (to my knowledge). I would not even worry about it.
